region.go

```diff
@@ -16,8 +16,6 @@ func GetRegion(zone string) (region string, err error) {
 		region = "sao"
 	case strings.HasPrefix(zone, "us-east"):
 		region = "us-east"
-	case strings.HasPrefix(zone, "tor"):
```

Comment:
why is this removed? I still see an entry here for tor - https://cloud.ibm.com/docs/power-iaas?topic=power-iaas-ibm-cloud-reg

Reply:
@Prajyot-Parab @dharaneeshvrd @Karthik-K-N am I missing something here?

Reply:
Its a [dup entry](https://github.com/ppc64le-cloud/powervs-utils/pull/15/files#diff-f53390ca7a4e1f4498b2ff8c143307da643ff451b0180b733ed294a9dcd84bc0L37), that's why she removed I guess.

Reply:
Yeah two entries were there for `tor`, so I have removed one